### DISCUSSION TITLE
Osu!Lazer not starting

### DISCUSSION THREAD

**Discussion by thepro2k** [ppy/osu#26994] [Q&A]
### Type

Game behaviour

### Bug description

Today I attempted to start the game as usual, but it won't launch. Upon looking at the task manager, I noticed that when I launch the game it shows  there for 1 or 2 seconds and then disappears.  Checking the event viewer show 2 application errors (attached bellow). 

I would appreciate if you could help me solve the problem with this information.

### Screenshots or videos

![image](https://github.com/ppy/osu/assets/108192718/6aeb5832-d82e-43bc-ba1a-4e30a61f2b7a)

![image](https://github.com/ppy/osu/assets/108192718/3f1d2ac3-989c-4648-9f68-0e54af99acca)

### Version

2024.131.0

### Logs

[1706908122.runtime.log](https://github.com/ppy/osu/files/14145667/1706908122.runtime.log)

**Comment by thepro2k** [comment]
After more testing,  i found that deleting the client.realm file allows my game to start as normal, but with default settings (I have have a backup of the old file)

**Comment by bdach** [comment]
I'm not sure why deleting your db would help. The error seen in the OP usually indicates driver level failure. The usual first thing we propose for that is updating graphics card drivers.
